evap/evaluation/templates/comment_visibility_buttons.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% load evaluation_templatetags %}
+
+<div class="btn-group" data-toggle="buttons">
+    {% for choice in form.comment_visibility %}
+        {% if choice.choice_value == "ALL" %}
+            {% trans "Will see all comments about the course and about all contributors." as tooltip %}
+        {% elif choice.choice_value == "COURSE" %}
+            {% trans "Will see own comments and comments about the course." as tooltip %}
+        {% elif choice.choice_value == "OWN" %}
+            {% trans "Default value. Will only see own comments." as tooltip %}
+        {% endif %}
+        {% if form.responsibility.value == "RESPONSIBLE" %}
+            {% include_choice_button form.comment_visibility choice False tooltip %}
+        {% else %}
+            {% include_choice_button form.comment_visibility choice True tooltip %}
```

Comment:
can you do something like `form.responsibility.value == "RESPONSIBLE" as disabled` and then use `disabled` as parameter? (i mixed up True and False but you get the idea)